daemon/volumes_unix.go

```diff
@@ -70,53 +68,6 @@ func sortMounts(m []execdriver.Mount) []execdriver.Mount {
 	return m
 }
 
-// migrateVolume links the contents of a volume created pre Docker 1.7
-// into the location expected by the local driver.
-// It creates a symlink from DOCKER_ROOT/vfs/dir/VOLUME_ID to DOCKER_ROOT/volumes/VOLUME_ID/_container_data.
-// It preserves the volume json configuration generated pre Docker 1.7 to be able to
-// downgrade from Docker 1.7 to Docker 1.6 without losing volume compatibility.
-func migrateVolume(id, vfs string) error {
```

Comment:
I don't think we should have removed this, but the fact that it's dead is likely the cause that a lot of people have had issues upgrading to 1.10... not sure where these were being called from before.

ping @calavera